modules/op-chains-generating-x509-secret.adoc

```diff
@@ -0,0 +1,55 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="chains-generating-x509-secret_{context}"]
+= Generating the x509 key pair using the TektonConfig CR
+
+To use the `x509` signing scheme for {tekton-chains} secrets, you must generate the `x509` key pair.
+
+You can generate the `x509` key pair by setting the `generateSigningSecret` field in the `TektonConfig` custom resource (CR) to `true`. 
+The {pipelines-title} Operator generates an `ecdsa` type key pair: `x509.pem` private key and `x509-pub.pem` public key, and stores them in the `signing-secrets` secret in the `openshift-pipelines` namespace.
+
+[WARNING]
+====
+If you set the `generateSigningSecret` field from `true` to `false`, the {pipelines-title} Operator overrides and empties the `signing-secrets` secret in the `openshift-pipelines` namespace. Therefore, ensure that you store the `x509-pub.pem` public key outside of the secret to protect it from the deletion and use it to verify later artifact attestations.
```

Comment:
```suggestion
If you set the `generateSigningSecret` field from `true` to `false`, the {pipelines-title} Operator overrides and empties any value in `signing-secrets`. Ensure that you store the `x509-pub.pem` public key outside of the secret to protect the key from deletion. The Operator can use the key at a later stage to verify artifact attestations.
```

Reply:
I applied the changes here except the last sentence as I believe my version is more minimalistic. I applied there your suggestion to replace "it" with "the key", therefore the second "it" no longer conveys double meaning and should be therefore okay to use :+1: